src/main/java/chess/dao/BoardDao.java

```diff
@@ -0,0 +1,75 @@
+package chess.dao;
+
+import chess.DBConnection;
+import chess.dto.BoardDto;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class BoardDao {
+
+    public int insertBoard(final BoardDto boardDto) {
```

Comment:
```suggestion
    public int insert(final BoardDto boardDto) {
```
BoardDao이니 모든 메서드명에서 Board는 생략해도 괜찮겠네요. SquareDao도 마찬가지구요.